### DISCUSSION TITLE
Welcome to vue-advanced-chat Discussions!

### DISCUSSION THREAD

**Discussion by antoine92190** [advanced-chat/advanced-chat-components#310] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by avenuesrini** [comment]
@antoine92190  let us know we can support you on Patreon

**Reply by antoine92190** [reply]
Thank you for you support! You can donate a one time or monthly donation by clicking on the Sponsor button on top of this page.
Or you can just click here: https://ko-fi.com/antoinedupont

**Comment by git-Majd** [comment]
does it support socket.io ? if it is canyou share to implement it

**Comment by webtech-soft** [comment]
Hi,

  How to get all rooms, when i click on cross icon on  room search box after search is performed. 
  please let me know as i am unable to handle this.

Thanks,
Gaurav.
